.github/workflows/dependencies_check.yml

```diff
@@ -29,13 +34,24 @@ jobs:
       - name: Generate ossindex report
         run: |
           mvn org.sonatype.ossindex.maven:ossindex-maven-plugin:audit \
-              org.sonatype.ossindex.maven:ossindex-maven-plugin:audit-aggregate \
-              -Dossindex.reportFile=$(pwd)/ossindex-report.json \
-              -Dossindex.fail=false
+              org.sonatype.ossindex.maven:ossindex-maven-plugin:audit-aggregate
 
-      - name: Report Security Issues
-        uses: exasol/python-toolbox/.github/actions/security-issues@main
-        with:
-          format: "maven"
-          command: "cat ossindex-report.json"
-          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This fails with a python error about noxfile. Deactivated for now as we still get notified about vulnerabilities